docs_manual/source/ext_tool_cyberduck.rst

```diff
@@ -0,0 +1,56 @@
+.. _ext_tool_cyberduck:
+
+Cyberduck
+^^^^^^^^^
+
+Cyberduck is a libre network storage browser for Mac and Windows.
+It can be used to access the iRODS filesystem on SODAR directly.
+
+Installation & Setup
+====================
+
+1. Download the Cyberduck installer from this website:
+https://cyberduck.io/download/
+
+.. note::
+    While Cyberduck is generally free to use, it will ask you to
+    donate to the project by making a one-time license key purchase.
+
+2. Download the SODAR iRODS connection profile:
+   :download:`iRODS (CUBI SODAR).cyberduckprofile <_static/ext_tool_cyberduck/iRODS (CUBI SODAR).cyberduckprofile>`
```

Comment:
We should replace ``sodar.bihealth.org`` in the profile with some placeholder host name and instruct the user to edit that to match the SODAR instance they used. This documentation can be read by users of other instances as well.

(..and before you say it: yes, there is other very CUBI/BIH specific stuff in the manual. We will also deal with that in time, see #1746)

Reply:
Will need to take new screenshots as well.